### ISSUE TITLE
property:taxonomicName

### ISSUE THREAD

**Issue by nielsklazenga** [tdwg/tcs2#2]
### taxonomicName (property)

|    |    |
|----|----|
| Label | Taxonomic Name |
| Definition | The Taxonomic Name for this Taxonomic Name Usage |
| Usage notes |  |
| Comments |  |
| Required | No |
| Repeatable | No |
| Constraints | Object |

#### Mapping

|    |    |
|----|----|
| TCS | DataSet/TaxonConcepts/TaxonConcept/Name |
| TDWG Ontology | [http://rs.tdwg.org/ontology/voc/TaxonConcept#hasName](https://github.com/tdwg/tnc/blob/master/tcs-docs/TaxonConcept.ttl#L65) |

**Comment by jar398** [comment]
Does a taxonomicName have to be something that is code compliant or approximately so?  For example, would it make sense to put an integer or a UUID in this field?

There are potential 'References' whose TNUs are easy to identify as keyed rows in a table, for example, but not by taxonomicName, either because they lack a taxonomicName (there is no associated code-compliant name given) or because namestrings are used ambiguously (e.g. hemihomonyms within a single Reference with no code designations, or that famous Girault example).  I'm wondering if these fit into TCS in any way.

(Am I asking this question in the correct place?)

**Comment by nielsklazenga** [comment]
Thanks @jar398 , yes, I think this is the right place and good question.

My opinion is that there should be no restrictions on what can be a taxonName in TCS. I might not like `taxonName`s with numbers, or indeed anything that is not code compliant, but I think decisions about how names are formatted is something for community implementations or guidelines, not for the standard.

As for whether the things you describe fit into TCS, I think they will have to. Even though I might not want these things in my own system, I still would like to be able to do taxon relationship assertions between them and the taxon concepts in my system.

**Comment by nielsklazenga** [comment]
@jar398 , see https://github.com/tdwg/tcs2/issues/42#issuecomment-775808273. I think what you are referring to might fall under OTU in that vocabulary?

**Comment by Archilegt** [comment]
About the label: Same issue as commented on "Taxon Concept". What makes sense to me is "Taxonomic Name". In Spanish "Nombre Taxonómico" is better than "Nombre de Taxon".

**Comment by Archilegt** [comment]
Ok, now to the point. This property is unnecessary, as its meaning is encoded in `scientificName` (http://rs.tdwg.org/dwc/terms/scientificName). The problem is that the definition of `scientificName` in Darwin Core is not ZooCode compliant, moreover, it is not even a definition.
  
In the ZooCode, what is a scientific name is regulated by article 1, and the number and type of words composing a scientific name are governed by articles 4 and 5.  Authorship and date of publication are not words composing the scientific names of animals.
  
Article 4. Names of taxa at ranks above the species group.
4.1. Names uninominal. The **scientific name** of a **taxon** of higher rank than the species group consists of **one word**.
  
Article 5. Principle of Binominal Nomenclature.
5.1. Names of species. The **scientific name** of a **species**, and not of a taxon of any other rank, is a combination of two names (**a binomen**), the first being the generic name and the second being the specific name.
  
5.2. Names of subspecies. The **scientific name** of a **subspecies** is a combination of three names (**a trinomen**, i.e. a binomen followed by a subspecific name).

It would be good to add documentation on how it is regulated in other codes, so that we can make progress in a new "Definition: The name of a taxon, conforming to Articles 1 and 4.1 or 5.1 or 5.2 of the ZooCode and... (add other articles/codes here)".

**Comment by nielsklazenga** [comment]
> About the label: Same issue as commented on "Taxon Concept". What makes sense to me is "Taxonomic Name". In Spanish "Nombre Taxonómico" is better than "Nombre de Taxon".

See https://github.com/tdwg/tcs2/issues/1#issuecomment-1026428075.

The important thing is that this is a name (or a label) for a taxon. Again, I think a 'taxonomic name' is something else.

**Comment by nielsklazenga** [comment]
> Ok, now to the point. This property is unnecessary, as its meaning is encoded in scientificName (http://rs.tdwg.org/dwc/terms/scientificName).

No, `scientificName` in Darwin Core is a string, while the `taxonName` property in TCS is an (Taxon Name) object (which can be referred to as a URI). If there is an equivalent in Darwin Core it is `scientificNameID`. There is a section in the Darwin Core RDF Guide about the problems with ...ID terms and why we cannot use them in most serializations.

`dwc:scientificName` is equivalent, I think, with `taxonomicNameString` (#16). We can discuss when we get there whether we borrow the Darwin Core term and change the label, or whether we create a new property.

**Comment by nielsklazenga** [comment]
Coming back to @jar398's question (https://github.com/tdwg/tcs2/issues/2#issuecomment-889227451);

> Does a taxonomicName have to be something that is code compliant or approximately so? For example, would it make sense to put an integer or a UUID in this field?

I will put it on the agenda for one of the coming meetings (not the immediately next one) of the TCS 2 Task Group, as it is a question we come back to every so often and I think it is important to see if we can find a solution we agree on (and if it is too hard we bump it down the track again).

I think that what is important here is that OTUs are Taxon Concepts (in the TDWG sense), not whether the string that is used as a label is a Taxon Name or not. So, I would like us to get off the idea that Taxon Concepts need to have taxon names. I think we should seriously consider (actually, adopt) @baskaufs's suggestion to use SKOS-XL labels.

The SKOS-XL labels are in addition to, not instead of, what we already have in TCS. The TCS Taxon Name would be understood as a subclass of `skosxl:Label`, although we cannot say so in the standard. I think this will allow us to use Taxon Name as intended in TCS 1, while at the same time recognising that not all Taxon Concepts have one (as TCS 1 did). Also, we would never have to argue again about whether phrase names and vernacular names are Taxon Names, as it does not matter (I think it never mattered); we all agree that they are all labels. Also, the `openbiodiv-o:TaxonomicConceptLabel` can be seen as a `skosxl:Label`, so we can have those without having a term for them in TCS.

In this line, I have also been thinking (quietly) that I would like to rename this property to `acceptedName` (that is when I had synonyms on my brain). Now I would also be happy with `scientificName`, which would nicely highlight the difference with the Darwin Core `scientificName`. [Strictly speaking, this would be a new term, as this property would be `skosxl:prefLabel`].

Just throwing this out there. I think this will work really well, but next week I might think differently.

**Comment by nielsklazenga** [comment]
Just noting that I have changed this the `taxonName` property to `acceptedName`. I think this makes it easier for people to understand what it means, as it is what we normally call it, and is better in the context of TCS. If people have strong objections (and have a good reason for them) it is changed back easily enough.

**Comment by nielsklazenga** [comment]
I think that for interoperability of TCS data sets, it would be really good if we make the `taxonName` required, so that people can rely on a `tcs:TaxonConcept` always having a `tcs:taxonName` and `tcs:accordingTo`. I cannot see a downside either, as people really wanting a string can just us a `skosxl:Label` with only a `skosxl:literalForm` or a `tcs:TaxonName` with only a `tcs:taxonNameString`.

This will be quite a lot of work to change, so I would like to hear people's opinions first.

Making it not required was probably a solo-action by me and I do not know what I was thinking then. Probably more about principle than usability.

**Comment by nielsklazenga** [comment]
I noticed that the usage still said the property is required, so I have gone ahead and cleaned up my mess.
